REINDEXING.md

```diff
@@ -0,0 +1,261 @@
+# Reindexing
+
+A reindex operation runs the source data from the [adapters](docs/adapters/README.md) through the pipeline causing it to be re-transformed / matched & merged as appropriate.
+
+## How to run a reindex
+
+Running a reindex will require:
+
+- Preparing and deploying a release
+- Terraforming a new pipeline using that release
+- Running the reindex script to feed the new pipeline
```

Comment:
I would maybe break this down into a checklist.

So up here you have a checklist that somebody who’s done reindexes can refer to, and further down you have more detailed explanations of what's going on.